Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -84,15 +92,29 @@ class BTAnalyticsService: Equatable {
                 return
             }
 
-            let postParameters = self.createAnalyticsEvent(config: configuration, sessionID: self.apiClient.metadata.sessionID, event: event)
-            self.http?.post("v1/tracking/batch/events", parameters: postParameters) { _, _, _ in }
+            // Send analytics events immediately if no timer is needed for unit tests
+            if self.timerInterval == 0 {
+                self.sendQueuedAnalyticsEvents(configuration: configuration)
+            } else {
+                DispatchQueue.main.asyncAfter(deadline: .now() + self.timerInterval) { [weak self] in
+                    self?.sendQueuedAnalyticsEvents(configuration: configuration)
```

Comment:
I changed the timer interval to 2 seconds and threw a print statement in this block. I would expect it to print to console every 2 seconds, but I'm not seeing that 👁️